paper/main.tex

```diff
@@ -856,17 +865,20 @@ \section{Evaluation of performance}\label{section:evaluation_of_performance}
 strategies exhibit some extortionate behaviour, but not constantly.
 ZDs are a set of strategies that are often envious as they try to exploit their
 opponents. The winners of the tournaments considered in this work are
-envious, but not as much as many ZDs. This highlights why TFT's early tournament
-success fails to generalize -- it never attempts to defect against a cooperating
-or exploitable opponent (e.g. Alternator). Moreover, many of the strategies in
-the library will not tolerate exploitation attempts. A clever strategy can
-achieve mutual cooperation with stronger strategies while also being able to
-exploit weaker strategies. This is why ZDs fail to appear in the tops
-ranks -- they try to exploit all opponents and cannot actively adapt back to
-mutual cooperation against stronger strategies, which requires more depth of memory.
-\footnote{Note that ZDs also tend to perform poorly in population games for
-a similar reason: they attempt to exploit other players using ZDs, failing to
-form a cooperative subpopulation. This makes them good invaders but poor resisters of invasion.}
+envious, but not as much as many ZDs.
+Though the exact interactions between the matches have not been recorded here,
+the work of~\cite{Harper2017} which introduced the trained strategies that
+appeared in the top ranked strategies of Section~\ref{section:top_performances}
+did. In~\cite{Harper2017} it was shown that clever strategies managed to achieve
+mutual cooperation with stronger strategies whilst exploiting the weaker
+strategies. This could potentially be what is happening with the clever winners
+of our analysis, and would explain the SSE distributions. This could also
+be the reason why ZDs fail to appear in the tops ranks -- they try to exploit
+all opponents and cannot actively adapt back to mutual cooperation against
+stronger strategies, which requires more depth of memory. \footnote{Note that
+ZDs also tend to perform poorly in population games for a similar reason: they
+attempt to exploit other players using ZDs, failing to form a cooperative
+subpopulation. This makes them good invaders but poor resisters of invasion.}
```

Comment:
```suggestion
stronger strategies, which requires more depth of memory. Note that
ZDs also tend to perform poorly in population games for a similar reason: they
attempt to exploit other players using ZDs, failing to form a cooperative
subpopulation. This makes them good invaders but poor resisters of invasion.
```

@Nikoleta-v3 if you're keen on a footnote then just ignore this suggestion.

Reply:
I don't know how I feel about footnotes. My supervisor never used them 😆

Reply:
Typically I use a footnote when I want to say something related but not go too far on a tangent. Some journals don't allow them though and I don't feel strongly about them in general. If adding a footnote back into the paragraph breaks the continuity then probably best to remove it.

Reply:
I don't mind it in the main body but just realised that this sentence needs a citation. Was there a particular paper you had in mind @marcharper?

Reply:
You could use our Moran paper?